packages/react/src/components/InAppMessaging/MessageLayout/MessageLayout.tsx

```diff
@@ -22,19 +25,21 @@ import {
   IMAGE_CONTAINER_CLASS,
   IMAGE_CONTAINER_TEST_ID,
   MESSAGE_LAYOUT_TEST_ID,
+  PRIMARY_BUTTON_TEST_ID,
+  SECONDARY_BUTTON_TEST_ID,
   TEXT_CONTAINER_CLASS,
   TEXT_CONTAINER_TEST_ID,
 } from './constants';
 import { MessageLayoutProps } from './types';
 import { getButtonModifier } from './utils';
 
+const isMessageButton = (button: unknown): button is MessageButtonProps =>
+  !isEmpty(button);
+
 export function MessageLayout({
   body,
   buttonSize,
-  hasButtons,
```

Comment:
For my understanding, is this component an internal only one that we call? Just because we are changing the API here. I assume it is, just confirming.

Reply:
Yeah it's internal only